src/schemas.js

```diff
@@ -78,7 +78,7 @@ const model = () => [
 const field = () => [
   {
     fieldType: "TextInput",
-    placeholder: "afield",
+    placeholder: "field_name_here",
```

Comment:
I would just use `field_name` for this one

Reply:
Makes sense for consistency, yes!